### PR TITLE
Aha/code cleanup

### PR DESCRIPTION
**Pull request by andyHa**

### CODE REVIEW

File: src/main/java/sirius/db/es/Elastic.java [hunk 1]
```diff
@@ -84,10 +86,15 @@
     @ConfigValue("elasticsearch.hosts")
     private String hosts;
 
+    @ConfigValue("elasticsearch.logQueryThreshold")
```
Comment (Tobsucht) [inline src/main/java/sirius/db/es/Elastic.java]: typo log <=> long

File: src/main/java/sirius/db/jdbc/schema/SchemaUpdateAction.java [hunk 2]
```diff
@@ -31,6 +31,11 @@
     private String realm;
     private volatile boolean executed;
 
+    /**
+     * Creates a new schema update action.
+     *
+     * @param realm the realm (database) for which this change is genrated
```
Comment (Tobsucht) [inline src/main/java/sirius/db/jdbc/schema/SchemaUpdateAction.java]: typo

File: src/main/java/sirius/db/mixing/annotations/Index.java [hunk 3]
```diff
@@ -41,6 +41,15 @@
      */
     String[] columns();
 
+    /**
+     * Contains a list of column settings to be used for <tt>Mongo DB</tt>.
+     * <p>
+     * Use {@link sirius.db.mongo.Mango#INDEX_ASCENDING} or {@link sirius.db.mongo.Mango#INDEX_DESCENDING} for
+     * commong use cases. Use appropriate strings determined by the Mongo DB documentation for special
```
Comment (Tobsucht) [inline src/main/java/sirius/db/mixing/annotations/Index.java]: typo

File: src/main/java/sirius/db/mongo/Mongo.java [hunk 4]
```diff
@@ -55,6 +57,10 @@
     @ConfigValue("mongo.db")
     private String dbName;
 
+    @ConfigValue("mongo.logQueryThreshold")
```
Comment (Tobsucht) [inline src/main/java/sirius/db/mongo/Mongo.java]: typo log <=> long

File: src/main/java/sirius/db/mongo/Mongo.java [hunk 5]
```diff
@@ -176,4 +183,20 @@ public Updater update() {
...
         return new Deleter(this);
     }
+
+    /**
+     * Returns the query log threshold in millis.
+     * <p>
+     * If the execution duration of a query is longer than this threshold, it is logged into
+     * {@link sirius.db.DB#SLOW_DB_LOG} for further analysis.
+     *
+     * @return the log thresold for queries in milliseconds
```
Comment (Tobsucht) [inline src/main/java/sirius/db/mongo/Mongo.java]: typo

File: src/main/resources/db_en.properties [hunk 6]
```diff
@@ -1,26 +1,26 @@
...
-SchemaTool.fkNeedsChange = The foreign key '${key}' of '${table}' needs to be changed.
-SchemaTool.fkUnused = The foreign key '${key}' of '${table}' is unused.
-SchemaTool.indexDoesNotExist = The index '${key}' of '${table}' does not exist.
-SchemaTool.indexNeedsChange = The index '${key}' of '${table}' needs to be changed.
-SchemaTool.indexUnused = The index '${key}' of '${table}' is unused.
-SchemaTool.pkChanged = The primary key of '${table}' need to be changed.
-SchemaTool.tableDoesNotExist = The table '${table}' does not exist.
-SchemaTool.tableUnused = The table '${table}' is unused.
-StringProperty.dataTruncation = The value '${value}' in the field '${field}' is too long with ${length} characters. A maximum of ${maxLength} characters is allowed.
+BaseEntityRefProperty.cannotDeleteEntityWithChild=The Entity cannot be deleted as the entity '${source}' holds one reference of type '${type}' in the field '${field}'.
```
Comment (Tobsucht) [inline src/main/resources/db_en.properties]: is this intended?

File: src/test/java/sirius/db/mongo/properties/MongoFilledSpec.groovy [hunk 7]
```diff
@@ -59,9 +62,16 @@ class MongoFilledSpec extends BaseSpecification {
              .count() == 1
 
         mango.select(MongoFilledEntity.class)
-             .where(QueryBuilder.FILTERS.exists(MongoFilledEntity.TEST_FIELD))
-             .queryList()
-             .every({e -> Strings.areEqual(e.getIdAsString(), fieldNotFilled.getIdAsString()) || Strings.areEqual(e.getIdAsString(), fieldFilled.getIdAsString()) })
+             .
```
Comment (Tobsucht) [inline src/test/java/sirius/db/mongo/properties/MongoFilledSpec.groovy]: formatting

File: src/main/java/sirius/db/es/Elastic.java (hunk 1 above)
Comment (andyHa) [reply src/main/java/sirius/db/es/Elastic.java]: no this is the threshold to determine if the query should be logged^^

Comment (sabieber) [reply src/main/java/sirius/db/es/Elastic.java]: Then the typo is in the field name

File: src/main/resources/component-db.conf [hunk 8]
```diff
@@ -81,6 +81,36 @@ health {
         redis-message-duration.gray = 25
         redis-message-duration.warning = 500
         redis-message-duration.error = 10000
+
+        # Number of calls against Elasticsearch
+        es-calls.gray = 100
+        es-calls.warning = 0
```
Comment (D3nnisH) [inline src/main/resources/component-db.conf]: `warning` oder `warn`?

**Pull request merged**